modules/ROOT/pages/dashboards/visualizations/barchart.adoc

```diff
@@ -0,0 +1,124 @@
+= Bar chart
+:description: The Neo4j Aura dashboard bar chart.
+
+A bar chart displays different categories and values in a bar layout.
+Choose the following:
+
+* *Category*: a text field. Categories are the bar labels.
+* *Value*: a numeric field. This determines the height of the bars.
+
+//* *Group*: A second optional text field. When grouping is enabled in the advanced settings, the group can be used to draw a stacked bar chart, with several groups per category.
+
+
+== Examples
+
+
+=== Simple bar chart
+
+A Cypher query for a bar chart which displays the number of people who acted in, directed or rated movies by title:
+
+[source,cypher]
+----
+MATCH (p:Person)-[e]->(m:Movie)
+RETURN m.title as Title, COUNT(p) as People ORDER BY People DESC LIMIT 20
+----
+
+// Screenshot
+
+////
+=== Stacked Bar Chart
+
+[source,cypher]
+----
+Match (p:Person)-[e]->(m:Movie)
+RETURN m.title AS Title, COUNT(p) as People, type(e) as Role ORDER BY Title, Role LIMIT 20
+----
+
+image::barstacked.png[Basic Table]
+////
+
+
+== Advanced settings
```

Comment:
will need to update these when we know the set of advanced settings we have for the preview release